<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2020-08-02T11:42:16-05:00" hostname="https://inspiration-computing-6472-dev-ed.cs91.my.salesforce.com" tests="1" failures="1"  errors="0"  time="1.53">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Aug 2, 2020 11:42 AM"/>
            <property name="testExecutionTime" value="1.53 s"/>
            <property name="testTotalTime" value="1.53 s"/>
            <property name="commandTime" value="14.28 s"/>
            <property name="hostname" value="https://inspiration-computing-6472-dev-ed.cs91.my.salesforce.com"/>
            <property name="orgId" value="00D2F000000AiZOUA0"/>
            <property name="username" value="test-lanivcw2dtmk@example.com"/>
            <property name="testRunId" value="7072F00001W0T0G"/>
            <property name="userId" value="0052F000006JcZ3QAK"/>
        </properties>
        <testcase name="shouldCreateOpportunityConnectionFromCustomValue" classname="formsConnectionCreateJobTest" time="1.53">
            <failure message="forms.fflib_SObjectUnitOfWork.UnitOfWorkException: SObject type Opportunity is not supported by this unit of work"><![CDATA[Class.forms.fflib_SObjectUnitOfWork: line 863, column 1
Class.forms.fflib_SObjectUnitOfWork.registerNew: line 315, column 1
Class.forms.fflib_SObjectUnitOfWork.registerUpsert: line 457, column 1
Class.forms.ResponsesServiceImpl.upsertConnectionSObject: line 306, column 1
Class.forms.ResponsesService.upsertConnectionSObject: line 60, column 1
Class.forms.ConnectionCreateJob.execute: line 62, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
